frontend-ssr/pages/api/user/check.js

```diff
@@ -0,0 +1,7 @@
+const {validate_jwt} =require("../../../utils/validation/middleware.validation")
```

Comment:
Not sure what this route does...🤔

Reply:
Change to `~/utils/...` . If it's not that DM me to talk about and I will update it

Reply:
Oooooh, now I saw this is an `api` route. Is it the `ping` one from the code I sent you? Or when is this route gonna be used?